present/json.go

```diff
@@ -52,7 +52,7 @@ func formatSpan(s *monkit.Span) interface{} {
 	js.Orphaned = s.Orphaned()
 	js.Args = make([]string, 0, len(s.Args()))
 	for _, arg := range s.Args() {
-		js.Args = append(js.Args, fmt.Sprint("%#v", arg))
+		js.Args = append(js.Args, fmt.Sprintf("%#v", arg))
```

Comment:
lol

Reply:
thanks go1.10 for automatically running go vet during go test! 👍